contracts/multisig-ism-metadata/src/main.sw

```diff
@@ -61,6 +48,55 @@ pub fn checkpoint_hash(origin: u32, mailbox: b256, root: b256, index: u32) -> b2
 }
 
 impl MultisigMetadata {
+    /// Constructs a new MultisigMetadata instance from packed bytes and a threshold.
```

Comment:
Note this is a slightly different encoding than the EVM one (bc of the 64 bit compact sigs vs the 65 bit ones in the EVM). The way things are written in the relayer already allows / expects the encoding to be chain-specific, so this doesn't cause any issues there